string name
int8 designation
AudioTopicInfo[] inputTopics
AudioTopicInfo[] outputTopics